### PR TITLE
setup.py: add the socket and service files to the data_files section

### PR DESCRIPTION
**Pull request by maurizio-lombardi**
Signed-off-by: Maurizio Lombardi <mlombard@redhat.com>

### CODE REVIEW

**Comment by maurizio-lombardi** [comment]
@pkalever @gonzoleeman While trying to create the rpm package for RHEL I realized that the socket and service files were not installed, preventing me from running the "systemctl enable targetclid.socket" command

File: setup.py [hunk 1]
```diff
@@ -34,6 +34,7 @@
                'scripts/targetcli',
                'daemon/targetclid'
               ],
+    data_files = [('lib/systemd/system', ['systemd/targetclid.socket', 'systemd/targetclid.service'])],
```
Comment (pkalever) [inline setup.py]: @maurizio-lombardi 

Please fix the path.

it should be:
```
 data_files = [('/lib/systemd/system', ['systemd/targetclid.socket', 'systemd/targetclid.service'])],
```

**Comment by pkalever** [comment]
@maurizio-lombardi IMO it will be nice to have a similar PR at rtslib for target.service

File: setup.py (hunk 1 above)
Comment (maurizio-lombardi) [reply setup.py]: Thanks, done. 
For some reasons it worked when I tested it on RHEL8.

**Comment by maurizio-lombardi** [comment]
> @maurizio-lombardi IMO it will be nice to have a similar PR at rtslib for target.service

Yeah, I will submit a PR later today

File: setup.py (hunk 1 above)
Comment (edacval) [reply setup.py]: `directory` path  in `data_files` must be relative !. Absolute path is deprecated and already broken on Arch, Gentoo , Fedora and RedHat > 8 . [Docs](https://docs.python.org/3/distutils/setupscript.html#installing-additional-files)

**Pull request merged**